express/index.cjs

```diff
@@ -346,9 +346,328 @@ app.post('/login/employer', async (req, res) => {
   }
 });
 
+
+//fetch jobs table from database
+app.get('/api/jobs', async (req, res) => {
+  try {
+    const [jobs] = await req.db.query('SELECT * FROM job');
+    res.status(200).json({ success: true, data: jobs });
+  } catch (err) {
+    console.error('Error fetching jobs:', err);
+    res.status(500).json({ success: false, error: 'Failed to fetch jobs' });
+  }
+});
+
+//forget password, should send email to link
+app.post('/forgot-password', async (req, res) => {
+  const newTime = new Date(Date.now());
+  writer.write(`email sent at : ${setTimestamp(newTime)} \ `);
```

Comment:
Malformed log, timestamp should always be first, sections separated by character `|`.